third-party/qthread/qthread-src/configure

```diff
@@ -3244,8 +3247,8 @@ test "$program_suffix" != NONE &&
 ac_script='s/[\\$]/&&/g;s/;s,x,x,$//'
 program_transform_name=`$as_echo "$program_transform_name" | sed "$ac_script"`
 
-# expand $ac_aux_dir to an absolute path
-am_aux_dir=`cd $ac_aux_dir && pwd`
+# Expand $ac_aux_dir to an absolute path.
```

Comment:
`configure` is a generated file (from `configure.ac`). Are these changes unrelated to hwloc due to using a different `autoconf` version that the one the Qthreads team uses? And should the hwloc-related changes in it actually be made to `configure.ac` instead?